### PR TITLE
Added tests

### PR DESCRIPTION
**Pull request by childen**
- More tests for invalid arguments
- Basic tests for valid arguments

I remove some of my messed up tests steps from duplicateArgumentsTest as I think this test should only test if duplicate arguments are detected and nothing else.

Includes Martins earlier commits. Auto merging should still work after his pull request is accepted.

### CODE REVIEW

File: src/final2/subtests/InvalidCommandLineArgumentsTest.java [hunk 1]
```diff
@@ -43,11 +45,49 @@ public void invalidArgumentFormatTest() {
...
+		errorTest("", Input.getFile(TASK_SHEET_INPUT_FILE_1), "rule=45-45-45-45-45", "rule=45-90-270-315-90");
 		errorTest("", Input.getFile(TASK_SHEET_INPUT_FILE_1), "speedup=1", "speedup=1");
 		errorTest("", Input.getFile(TASK_SHEET_INPUT_FILE_1), "speedup=1", "speedup=5");
 	}
+	
+	/**
+     * Asserts that the tested class prints an error message if an invalid rule command is passed.
+     */
+    @Test
+    public void invalidRuleArgumentsTest() {
```
Comment (pcworld) [inline src/final2/subtests/InvalidCommandLineArgumentsTest.java:58]: add a "too big number" test to this, too?
Edit: E.g.
`errorTest("quit", Input.getFile(TASK_SHEET_INPUT_FILE_1), "rule=270-90-99999999999999999999999999999999999999999-315-45");`

Comment (childen) [inline src/final2/subtests/InvalidCommandLineArgumentsTest.java:58]: added

**Comment by pcworld** [comment]
> Task sheet does not ask for a specific exit status if there is no error and quit is called

I remember reading something on ILIAS that Praktomat expects 0 exit code for successful runs.
Maybe someone can try to upload a solution with exit code 1 and see if Praktomat complains.

**Comment by jGleitz** [comment]
@pcworld: Even if the Praktomat would accept it, I'm quite sure you'd lose point for exiting with anything but `0` after a successful run. Therefore, I'll keep testing it.

File: .classpath [hunk 2]
```diff
@@ -1,7 +1,7 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<classpath>
-	<classpathentry kind="src" path="src"/>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
-	<classpathentry kind="con" path="org.eclipse.jdt.junit.JUNIT_CONTAINER/4"/>
-	<classpathentry kind="output" path="bin"/>
-</classpath>
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jGleitz) [inline .classpath:1]: Why are we even including `.classpath`? I guess I forgot to add it to `.gitignore`. Or are there any reasons to keep it?

Comment (pcworld) [inline .classpath:1]: If we get rid of it, we should include javadoc.css too
Edit: I mean delete it, adding it to .gitignore is probably not necessary

**Pull request merged**